### ISSUE TITLE
A couple of issues related to Blog

### ISSUE THREAD

**Issue by tjayrush** [TrueBlocks/trueblocks-docs#35]
**Issue 1**

It looks like the blog has a different margins / spacing from the 'docs' section. Like the list is too far over to the right and possible the Categories are over to the right.

**Issue 2**

The space between the blog entries at the topmost blog page is too wide. I'd prefer much closer line spacing.

**Issue 3**

I wonder if we can see some more data on the list of all blog posts on the main blog page. Like the 'middle' page, the subtitle could be displayed or maybe the data (or even the tags?)

**Issue 4**

The 'middle' page has not navigation (as you know).

**Comment by MattDodsonEnglish** [comment]
I agree with all these, and I'm going to work on fixing them this week.

I've probably hit a ceiling with my strategy of "just change random things and hope for the best," so I may need to go back a step to learn the hugo templating language better.

**Comment by tjayrush** [comment]
Changing things randomly may not be an excellent strategy, but if we looked carefully at my commit histories, you might convince me that's not true.

**Comment by MattDodsonEnglish** [comment]
Update on this issue

## Issue 1
> It looks like the blog has a different margins / spacing from the 'docs' section. Like the list is too far over to the right and possible the Categories are over to the right.

This is fixed. Well, there is no side category on the top level `/blog` path anymore. Rather, there's just 4 categories. Screenshot, `trueblocks.io/blog`,  2021-09-10
![image](https://user-images.githubusercontent.com/47385188/132850875-9236a63e-5be0-4d2e-bd14-f8b9464e2787.png)

The actual blog posts have a "Tags" side bar. I think it looks fine.  However, the "/tags"  list is a bit brutalistic. See issue 4.

![image](https://user-images.githubusercontent.com/47385188/132855997-ecfae256-5c39-4591-a9ff-b93adc3a71e9.png)


## Issue 2
> The space between the blog entries at the topmost blog page is too wide. I'd prefer much closer line spacing.

I remember I agreed with this, but I don't remember what this was about anymore! Is this is fixed `*in absentia* of example?

Anyway, you see how /blog looks, here's `trueblocks.io/blog/ethereum-articles`, screenshot 2021-09-10:
![image](https://user-images.githubusercontent.com/47385188/132851431-2a20e696-81fb-4538-92fc-d0fb32948205.png)

The issues that remain: 
* Maybe we can add pagination to the categories (e.g. ten posts per page)
* Each topic card could have more data, e.g. publication data and contributor. This is the _hardest problem_, see Issue 3.

## Issue 3

> I wonder if we can see some more data on the list of all blog posts on the main blog page. Like the 'middle' page, the subtitle could be displayed or maybe the data (or even the tags?)

To these cards, we can add any data that exists in the blog posts front matter, e.g. `lead` `date` `description`, and it might be possible to add custom parameters too. However, this is a little bit tricky. I'll explain.

In Hugo, each section can have a `list.html` page. In our example, `blog`is a category, and the page you see at `/blog` is the list.html. 

However, `explainers`, `ethereum-articles`, etc are sub-sections of the blog section, so they inherit the properties of the blog's list.html. So for now, the cards at `/blog` and `blog/ethereum-articles` have the same properties. So, If I added more data, it might look non-sensical in one of the levels. For example "0 minute reading time" on the `/blog` and "0 articles inside" on `/blog/ethereum-articles`

I am not sure whether it is possible to make a sub-list.html for a sub-section. I will research. If it is possible, this fix is easy--add the right data to the right card. If it is not possible, we will have to make a decision about what's "good enough" for both card levels (or wait for someone smarter to figure something else out).

Does that make sense? 

## Issue 4

The 'middle' page has not navigation (as you know). I think that is mostly fixed, see [issue 1](#issue-1). The only exception is the /tags list. That currently mimics `/blog` list, but there's nothing but the tag name.
![image](https://user-images.githubusercontent.com/47385188/132856864-44a2bca8-7b06-42d5-a61b-4def6e1d4e96.png)


 I think this is a small issue.  _Maybe_, I can replace that with its own tags-list.html to show a tag cloud. But that has all the same problems described in issue 3.

**Comment by MattDodsonEnglish** [comment]
Also, didn't even notice until now that it says  " Tag:"Tags". Just pushed 04ce45c46fee9f44c21f69dc2fbbe1ca0c2acdac to fix that so it says Tags: "All".

**Comment by MattDodsonEnglish** [comment]
Think we can close this with commits: https://github.com/TrueBlocks/trueblocks-docs/compare/814c7f4db084...5f31dcfe3ab4

Although, it's still not perfect
